cli/server/server.go

```diff
@@ -52,10 +52,6 @@ func NewCommands() []cli.Command {
 	var cfgCountInFlags = make([]cli.Flag, len(cfgWithCountFlags))
 	copy(cfgCountInFlags, cfgWithCountFlags)
 	cfgCountInFlags = append(cfgCountInFlags,
-		cli.UintFlag{
-			Name:  "skip, s",
-			Usage: "number of blocks to skip (default: 0)",
```

Comment:
I've used it for debugging some blocks (reapplying the same block at height X again and again from non-incremental dump). If we're removing it then skipping must be automatic for all cases (which makes sense, but probably we should log something like "dump starts as X, chain is at Y, skipping Z blocks).

Reply:
Added log. Debugging is exactly where this offset calculation is annoying. Twice so with incremental dumps.